__tests__/src/integration.spec.js

```diff
@@ -15,14 +15,37 @@
 const fs = require('fs');
 const path = require('path');
 const toMatchImageSnapshot = require('../../src').toMatchImageSnapshot;
+const rimraf = require('rimraf');
 
 expect.extend({ toMatchImageSnapshot });
 
 describe('integration tests', () => {
   const imagePath = path.resolve(__dirname, '../stubs', 'TestImage.png');
-  const data = fs.readFileSync(imagePath);
+  const imageData = fs.readFileSync(imagePath);
+
+  beforeAll(() => {
+    rimraf.sync('./__tests__/src/__image_snapshots__');
+  });
+
+  it('writes a snapshot with no error.', () => {
+    expect(() => expect(imageData).toMatchImageSnapshot({ customSnapshotIdentifier: 'integration' })).not.toThrowError();
+  });
 
   it('matches an identical snapshot.', () => {
-    expect(data).toMatchImageSnapshot();
+    expect(() => expect(imageData).toMatchImageSnapshot({ customSnapshotIdentifier: 'integration' })).not.toThrowError();
+  });
+
+  it('fails with a different snapshot.', () => {
+    const failImagePath = path.resolve(__dirname, '../stubs', 'TestImageFailure.png');
+    const failImageData = fs.readFileSync(failImagePath);
+
+    expect(() => expect(failImageData).toMatchImageSnapshot({ customSnapshotIdentifier: 'integration' })).toThrowError();
+  });
+
+  it('fails gracefully with a different snapshot.', () => {
```

Comment:
Can you rename this test to something more meaningful like `fails gracefully with a differently sized image`?